src/Components/Clash/ClashBox/ClashBox.css

```diff
@@ -0,0 +1,5 @@
+#ClashBoxStyle {
```

Comment:
I strongly suggest adding some mobile specific styling here, the current layout works really nicely down to tablet sizes, but not so well on mobile.

Maybe put the clash topping cards in a single stacked column or something?

Reply:
We'll get onto that.